.github/workflows/ios.yml

```diff
@@ -12,69 +12,129 @@ jobs:
     runs-on: macOS-latest
     steps:
       - uses: actions/checkout@v1
+
+      - name: List available versions of Xcode
+        run: ls -n /Applications/ | grep Xcode*
+
       - uses: actions/setup-node@v1
         with:
-          node-version: '16.1.0'
+          node-version: '14.16.1'
+
       - name: Install javascript packages
         uses: bahmutov/npm-install@v1
+
       - uses: ruby/setup-ruby@v1
         with:
           ruby-version: '2.7.3'
+
       - name: Install Fastlane
         run: bundle install
+
       - name: Install cocoapods
         run: gem install xcpretty cocoapods
+
+      - name: TEMP manually run pod install
```

Comment:
Does `TEMP` indicate here that you actually planned to remove these steps?

Reply:
yes, done!